### PR TITLE
Added security.txt endpoint

### PR DESCRIPTION
**Pull request by taskinoz**
Added a `/.well-known/security.txt` endpoint which list the 3 major repos:
- NorthstarMasterServer
- NorthstarLauncher
- NorthstarMods

Solves issue #57

### CODE REVIEW

**Comment by pg9182** [comment]
huntr.dev?

**Comment by taskinoz** [comment]
We could possibly leave discord usernames as contact info for people to get a hold of

**Comment by GeckoEidechse** [comment]
> We could possibly leave discord usernames as contact info for people to get a hold of

That could also work.

File: web/.well-known/security.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Northstar Master Server
+https://github.com/R2Northstar/NorthstarMasterServer
+
+# Northstar Launcher
+https://github.com/R2Northstar/NorthstarLauncher
+
+# Northstar Game Scripts
+https://github.com/R2Northstar/NorthstarMods
+
+# Or notrify one of our developers on Discord
```
Comment (GeckoEidechse) [inline web/.well-known/security.txt:14]:
```suggestion
# Or notify one of our developers on Discord
```
 
Typo :P

**Comment by pg9182** [comment]
If you are planning to continue working on this, it needs to be re-submitted against [R2Northstar/NorthstarTF](https://github.com/R2Northstar/NorthstarTF).

**Pull request closed without merging**